junebug/api.py

```diff
@@ -175,7 +175,7 @@ def get_channel(self, request, channel_id):
                 'config': {'type': 'object'},
                 'metadata': {'type': 'object'},
                 'status_url': {'type': 'string'},
-                'mo_url': {'type': 'string'},
+                'mo_url': {'type': ['string', 'null']},
```

Comment:
I think we want to do `status_url` as well, I think it's the only other one that we could accept `null` on.